### PR TITLE
december release notes and blog post

### PR DESCRIPTION
**Pull request by danabauer**
## What's in this PR? 
- December 2024 release notes
- updating release date in config
- transportation blog post

### Docs Preview:

Release notes: https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/142/release/latest/
Blog post: https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/142/blog/

View all staged runs:
https://github.com/OvertureMaps/docs/actions/workflows/publish-pr-to-staging.yml

### CODE REVIEW

File: release-blog/2024/2024-12-18.0.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+### Buildings
+- made incremental updates
+- hit an all-time high of 2.3 billion building features
+
+### Divisions
+- added a `class` property to explicitly call out the type of locality, e.g. hamlet, city
+- added new properties: `is_land`, `is_territorial`, and `prominence`
+- fixed issue with duplicate data in Tuvalu
+- added TIGER locality data for the United States
```
Comment (stepps00) [inline release-blog/2024/2024-12-18.0.mdx:64]: This is coming in Q4 - hasn't landed yet.

File: blog/2024-12-18-transportation-to-ga.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- In the [November release](https://docs.overturemaps.org/release/2024-11-13.0/), we removed `lanes` from the schema to eliminate a significant trust violation. The `lanes` property has existed in the schema for months but we never populated it with data in our releases, which is a poor experience for our community. We intend to redesign `lanes` in the coming months.
+
+With these changes, we are confident we’ve met our requirements for general availability (GA). We'll continue refining our schema and improving our data throughout 2025. We encourage our community to keep the feedback coming, particularly around data quality and our core use cases of map display, routing, and matching data via GERS.
+
+Here are some specific ways you can help:
+
+1. Examine transportation data in your areas of interest and compare it to local sources of data. Here are some suggestions for downloading data using Overture's Python CLI.
```
Comment (stepps00) [inline blog/2024-12-18-transportation-to-ga.mdx:26]: Worth a link to the CLI tool? https://docs.overturemaps.org/getting-data/overturemaps-py/

File: release-blog/2024/2024-12-18.0.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+:::
+
+### Addresses
+- reached a milestone of 400 million address records
+- added new data for:
+    - Czechia
+    - Liechtenstein
+    - Uruguay
+    - various counties in California
+- replaced NAD data in Massachusetts with open data from MassGIS
```
Comment (jwass) [inline release-blog/2024/2024-12-18.0.mdx:49]: @stepps00 We got a few more datasets in California now too?

**Review by jwass — APPROVED** [review]

**Review by stepps00 — CHANGES_REQUESTED** [review]
Requesting one change for divisions. Looks great!

File: release-blog/2024/2024-12-18.0.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- added `bathymetry` feature type with 60,000 new features
+- added new classes to `land`, `land_use`, and `water` feature types, including `fatbike`, `hike`, and `nordic`
+
+### Buildings
+- made incremental updates
+- hit an all-time high of 2.3 billion building features
+
+### Divisions
+- added a `class` property to explicitly call out the type of locality, e.g. hamlet, city
+- added new properties: `is_land`, `is_territorial`, and `prominence`
```
Comment (stepps00) [inline release-blog/2024/2024-12-18.0.mdx:62]: We can also remove this line, too - this will land in January.

**Pull request merged**